strmangle/inflect.go

```diff
@@ -94,6 +94,7 @@ func newBoilRuleset() *inflect.Ruleset {
 	rs.AddSingular("s", "")
 	rs.AddSingular("ss", "ss")
 	rs.AddSingular("as", "as")
+	rs.AddSingularExact("areas", "area", true)
```

Comment:
I hate to make a comment on this PR that requires changes. But this should be with the other `AddSingularExact` calls.